src/Bicep.Core/Semantics/SemanticModelHelper.cs

```diff
@@ -2,10 +2,15 @@
 // Licensed under the MIT License.
 
 using System;
+using System.IO;
 using System.Linq;
 using Bicep.Core.Semantics.Namespaces;
 using Bicep.Core.Syntax;
 using Bicep.Core.TypeSystem;
+using System.Text.Json;
+using Bicep.Core.Workspaces;
+using Bicep.Core.Json;
+using Bicep.Core.Extensions;
```

Comment:
```suggestion
using System.Linq;
using Bicep.Core.Semantics.Namespaces;
using Bicep.Core.Syntax;
using Bicep.Core.TypeSystem;
```

(minor) These look to be unused. I'm guessing these came from auto-imports when working in this file and then code in this file got refactored to another file?

Reply:
6ff824b0ba71a758fb1148753e4d80c87f63d102